src/user/repositories/user.repository.ts

```diff
@@ -1,6 +1,16 @@
+import { NotFoundException } from '@nestjs/common';
 import { EntityRepository, Repository } from 'typeorm';
 
 import { User } from '../entities/user.entity';
 
 @EntityRepository(User)
-export class UserRepository extends Repository<User> {}
+export class UserRepository extends Repository<User> {
+  async getById(id: number): Promise<User> {
+    const room = await this.findOne(id);
```

Comment:
```suggestion
    const user = await this.findOne(id);
```

Reply:
Thanks for the catch. :) 
I have updated it.